### ISSUE TITLE
Setup database not working via the app

### ISSUE THREAD

**Issue by rajurh** [qxsch/ContosoHotel#4]
Setup database doesn't work in few instances. It resulted in the below error.

If we populate the data into DB directly the same gets reflected in the front end, so definitely its something to do with application.

![image](https://github.com/user-attachments/assets/a053de7f-dd22-4a72-8de8-33b60b165da5)

**Comment by qxsch** [comment]
@rajurh 

It seems, that the api endpoint does return html insted of json.

since the api endpoint in the code just returns json by design, this is a clear indication, that something in between is intercepting (f.e. Firewall, reverse proxy, …) and generating the failure.

can you open up the dev tools in the browser and then share the response from the networking tab to get more insights what is actually failing?

**Comment by rajurh** [comment]
Sure, here are the additional details.

I am running it as a container locally and connecting to Azure Postgres SQL DB. 

Network logs
Request payload
{"drop_schema":false,"create_schema":false,"populate_data":true}

Response payload
<html>
  <head>
    <title>Internal Server Error</title>
  </head>
  <body>
    <h1><p>Internal Server Error</p></h1>
    
  </body>
</html>

![image](https://github.com/user-attachments/assets/70e8a7f7-1112-4127-ab77-ce28bcd673d3)

**Comment by qxsch** [comment]
The response is just showing internal server error?

what are you using?

did you run it locally in docker desktop?
does it show in docker desktop the same error?

**Comment by qxsch** [comment]
It could also be, that your runtime environment kills the request after 30s.
Try to increase processing timeout of the runtime.

**Comment by rajurh** [comment]
Yes I am running in docker desktop and above errors are from it. 

How and where do we make code change to increase the processing timeout of the run time please?

**Comment by qxsch** [comment]
I think, currently we haven’t identified the root cause here.

**Comment by claick-oliveira** [comment]
Edit the file startup.sh to startup.sh gunicorn --bind=0.0.0.0 --workers=4 startup:app --timeout 180
